.mailmap

```diff
@@ -95,6 +95,8 @@ Naresh Jagadeesan <naresh.naresh000@gmail.com> <37257700+Infinage@users.noreply.
 
 Nithin Katta <88046362+nithinkatta@users.noreply.github.com> KATTA NAGA NITHIN
 
+Neeraj Pathak <neerajrpathak710@gmail.com> Neerajpathak07
```

Comment:
@Neerajpathak07  I advise against making this change in this PR. Your PR should be limited to the specific purpose of this PR. If the mailmap file should be updated, you should send a separate PR.

Reply:
ok sure I'll remove it as of now